hi-ml-histopathology/Makefile

```diff
@@ -1,3 +1,10 @@
+# Make commands for the toolbox users
+
+# Create a Conda environment for this folder only
+env:
+	conda env create --file environment.yml
+	pip install -r ../test_requirements.txt
```

Comment:
Will this install all necessary dependencies, including AML SDK and friends?

Reply:
yes